modules/modules.nix

```
@@ -158,6 +158,7 @@ let
     (loadModule ./services/password-store-sync.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/pasystray.nix { })
     (loadModule ./services/picom.nix { })
+    (loadModule ./services/plan9port.nix { })
```

Since the module uses systemd:
```suggestion
    (loadModule ./services/plan9port.nix { condition = hostPlatform.isLinux; })
```